pkg/frame/inplace/read.go

```diff
@@ -0,0 +1,29 @@
+// Package inplace contains code for handling UnprocessedValueData
+package inplace
+
+// ReadUnprocessedValueData read the value of Dicom image directly
+// from the byte array of PixelData.UnprocessedValueData with given frame ID.
+// This ease the memory usage of reading DICOM image.
+func ReadUnprocessedValueData(info *PixelDataMetadata, unprocessedValueData []byte, frameIndex int) ([][]int, error) {
```

Comment:
Since this returns [][]int, this will still have the same issue of expanding the memory from the smaller int to the 64-bit int right? Is the main place this saves memory in the fact that it only does one frame at a time?

Reply:
> Is the main place this saves memory in the fact that it only does one frame at a time

Yes, most of the DICOM with the issues contains multiple frames.

But, I think I would change to a new structure of storing data. 

B/c users need to adapt to new way of read/write pixel data anyway.

Reply:
Changed at 16701faf01880474451f742d3a8c7b4aff370b61